#1870 - On reset apply the filter values from controls

dnahodil opened the pull request:
Fixes #1870

jkburges commented on web-app/js/portal/details/NcWmsPanel.js:
```diff
@@ -383,6 +383,7 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
         this._updateTimeRangeLabel();
 
         this._setLayerSubsetExtent();
+        this._applyFilterValuesFromMap();
```
I think this is a bit confusing.  Really, what we want to express at this level is `_applyFilterValuesToCollection`.  This method should in turn do the bit about getting the constraint from the map (it shouldn't really be getting it from *there*, but that's a different story).

dnahodil commented:
Changes made. I think I addressed your comment, if you meant that the important thing is that the filters are being applied *to the collection* and not the the filters were *coming from the map*.
